natlas-server/app/admin/routes.py

```diff
@@ -427,10 +428,10 @@ def deleteHost(ip):
 	if delForm.validate_on_submit():
 		deleted = current_app.elastic.delete_host(ip)
 		if deleted > 0:
-			flash("Successfully deleted host %s" % ip, "success")
-			return redirect(url_for('main.search'))
+			flash(f"Successfully deleted {deleted - 1 if deleted > 1 else deleted} scans for {ip}", "success")
```

Comment:
The syntax is a little strange (I think it's fairly unique to python?), but basically we do `deleted - 1` only if `deleted > 1` else we just display `deleted` It's basically just a way to do inline conditionals in python instead of having to do

```
if deleted > 1:
    deleted = deleted - 1
flash(........
```